### DISCUSSION TITLE
How do I launch the program?

### DISCUSSION THREAD

**Discussion by Thehummingbee** [MaxRobinsonTheGreat/EvolutionSimulatorV2#136] [General]
I promise I'm smart.

I downloaded the program, but don't see any exe to click on. Could someone please tell me what to click? 
Thank you in advance.

**Comment by MaxRobinsonTheGreat** [comment]
If you just want to play the game, you can go to https://thelifeengine.net

If you want to edit the source code and run it yourself, you have to install npm, download this repository, and then in the downloaded folder run the command line `npm install`, then `npm run build`, then open the `./dist/index.html` file.
